tuareg.el

```diff
@@ -737,8 +739,8 @@ delimiting the region of interest. "
         (setq group-number (+ group-number 1 re-ngroups))))
     (let ((combined-re (mapconcat (lambda (re) (concat "\\(" re "\\)"))
                                   (nreverse regexps) "\\|"))
-          (begin (gensym "begin"))
-          (end (gensym "end")))
+          (begin (make-symbol "begin"))
```

Comment:
It would be better to define `tuareg--gensym` that would be aliased to `gensym` is it exists and redefine it if not.  There is an `end` that will conflict in the body it is applied to.

Reply:
Can be done, but is it worth the trouble? `make-symbol` works just as well; I only used `gensym` during development to make it easier to debug my stupid mistakes. (There is no actual symbol clash; it only looks like it when looking at the expanded macro code.)